recipes/default.rb

```diff
@@ -51,3 +51,10 @@
   user node['timesync']['user']
   action :start_or_graceful_reload
 end
+
+directory '/opt/timesync/source' do
```

Comment:
So it looks like the real problem is the fact that the ``nodejs_webapp`` resource doesn't properly set the user to checkout the git repo as. I'd recommend we fix that properly and then setting the source directory wouldn't be a problem.